pom.xml

```diff
@@ -22,7 +22,7 @@
     <properties>
         <attachVersion>4.0.2</attachVersion>
```

Comment:
Latest Attach release is 4.0.3

Reply:
different issue

Reply:
It is the same issue, as Attach 4.0.3 includes the required json files for reflection